src/Paramore.Brighter.MessagingGateway.Kafka/KafkaMessageConsumer.cs

```diff
@@ -535,7 +532,7 @@ private void Close()
             catch (Exception ex)
             {
                 //this may happen if the offset is already committed
-                s_logger.LogDebug("Error committing the current offset to Kakfa before closing: {ErrorMessage}", ex.Message);
+                s_logger.LogDebug("Error committing the current offset to Kafka before closing: {ErrorMessage}", ex.Message);
```

Comment:
I'm glad it's not just me that spells it Kakfa at speed, my fingers don't want to do it the right way